lib/public/utilities/fetch/buildUrl.js

```diff
@@ -74,22 +73,24 @@ export const buildUrl = (baseURL, parameters) => {
         }
     };
 
+    const [origin, existingParameters] = baseURL.split('?');
```

Comment:
It is not origin. For example `http://example.com/route` has `http://example.com` as origin, not the `/route`

Reply:
fixed